### PR TITLE
Added basic overall dashboard with links to each class stats page

### PR DESCRIPTION
**Pull request by pjsier**
Added a dashboard page at `admin/dashboard` that currently includes charts for age and gender, percentage overall attendance, and a listing of all past classes with links to their check-in pages which have more details stats on demographics and attendance.

Planning on adding additional demographics.

### CODE REVIEW

File: coderdojochi/views.py [hunk 1]
```diff
@@ -1272,6 +1272,46 @@ def session_announce(request, session_id):
...
+    gender_count = sorted(dict(gender_count).items(), key=operator.itemgetter(1))
+
+    # Ages
+    ages = sorted(list(e.student.get_age() for e in checked_in_past_orders))
+    age_count = list(Counter(ages).iteritems())
+    age_count = sorted(dict(age_count).items(), key=operator.itemgetter(1))
+
+    # Average Age
+    average_age = 0
+    average_age = int(round(sum(ages) / float(len(ages))))
```
Comment (karbassi) [inline coderdojochi/views.py]: Why is `average_age` set to `0` and then overwritten?

**Comment by pjsier** [comment]
@karbassi good catch on the `average_age` assignment. I'd left it in from the null check that's in the session check in view, but given there will always be at least one it's unnecessary here. Forgot to mention this is in reference to #72

**Comment by karbassi** [comment]
@pjsier Could you create a fixture with some sample data so we can test it?

**Comment by pjsier** [comment]
@karbassi Just added some fixtures that should work!

**Pull request merged**